pkg/registry/lock/registry.go

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
Carrying over comment by @wojtek-t from previously closed PR:

Can you please be consistent with all the other resources:
- registry.go file contains "Registry" interface (Create/Update/Delete/List/Get/Watch methods)
- rest.go file contains "strategy" (if needed)
- etcd/etcd.go contains "REST" definition

I agree that it's very misleading (we are going to change it). But for consistency it's much better to have everything to be the same.